docs/src/css/custom.css

```diff
@@ -72,3 +72,17 @@ thead {
 .hide-expandable-sidebar ul {
     display: none !important;
 }
+
+
+.link_t2Qp {
```

Comment:
@yaruno might be wise to add a comment to these `link_t2Qp`s explaining what they relate to